common/ast/src/ast/expr.rs

```diff
@@ -689,6 +693,17 @@ impl<'a> Display for Expr<'a> {
                 write_comma_separated_list(f, exprs)?;
                 write!(f, "]")?;
             }
+            Expr::Interval { expr, unit, .. } => {
+                write!(f, "INTERVAL({expr} {unit})")?;
```

Comment:
seems we should not need parentheses.

Reply:
ok. actually parentheses is more beautiful